spec/requests/api/v1/docs/tags_spec.rb

```diff
@@ -0,0 +1,69 @@
+require "rails_helper"
+require "swagger_helper"
+
+# rubocop:disable RSpec/EmptyExampleGroup
+# rubocop:disable RSpec/VariableName
+
+RSpec.describe "Api::V1::Docs::Tags" do
+  let(:Accept) { "application/vnd.forem.api-v1+json" }
+  let(:api_secret) { create(:api_secret) }
+  let(:user) { api_secret.user }
+
+  describe "GET /tags" do
+    before do
+      user.add_role(:admin)
+    end
+
+    path "/api/follows/tags" do
+      get "Followed Tags" do
+        tags "tags"
+        description "This endpoint allows the client to retrieve a list of the tags they follow."
+        operationId "getFollowedTags"
+        produces "application/json"
+
+        response "401", "Unauthorized" do
+          let(:"api-key") { nil }
+          add_examples
+
+          run_test!
+        end
+
+        response "200", "A list of followed tags" do
+          let(:"api-key") { api_secret.secret }
+          schema  type: :array,
+                  items: { "$ref": "#/components/schemas/FollowedTag" }
```

Comment:
@rt4914 I'm not seeing a `FollowedTag` schema in the [swagger_helper](https://github.com/forem/forem/blob/790870b4597da76485ff3d38a8f4796015b6a413/spec/swagger_helper.rb#L263)? Does this reference a schema from a different file? 

The same comment applies to `"#/components/schemas/Tag"` below?

Reply:
I had actually referenced it from https://github.com/forem/forem-docs/blob/b5ffacaf77b9ed09df7e2d6029fd25d17c246b7c/api_v0.yml#L2617

I was not aware about this `swagger_helper`. Considering it does not contain that scheme: (a.) should I create a schema in that or I should remove the `scheme` from `tags_spec.rb` ?

Reply:
I agree with what Ridhwana mentions, and I think the best path forward is adding the `Tag` scheme in swagger_helper.rb (similar to [ArticleFlareTag in there](https://github.com/forem/forem/blob/c94148c987661d9d56a4d8e6a0b565a7a98594a6/spec/swagger_helper.rb#L159)), but open to hearing out what you would prefer @Ridhwana in case you feel differently 🙂 

With the schema linked here swagger would then generate a response with it (array of `Tags`). You could reuse the same schema for both of them (`Tag`), since the result is just an example on the docs and don't see a gain in having two exact schemas for these two separate endpoints.

When the link is working properly you won't see this error if you copy&paste the JSON output into https://editor.swagger.io for preview

<img width="1375" alt="Screen Shot 2023-01-03 at 13 20 32" src="https://user-images.githubusercontent.com/6045239/210426367-0ede0c8f-a3d8-43b6-a438-88ec5ee20b9a.png">

Reply:
> I think the best path forward is adding the Tag scheme in swagger_helper.rb (similar to [ArticleFlareTag in there](https://github.com/forem/forem/blob/c94148c987661d9d56a4d8e6a0b565a7a98594a6/spec/swagger_helper.rb#L159)), but open to hearing out what you would prefer @Ridhwana in case you feel differently 🙂

+ 1 on this.

Reply:
I have made the required changes.

@fdocr The schema for [FollowedTag](https://github.com/forem/forem-docs/blob/b5ffacaf77b9ed09df7e2d6029fd25d17c246b7c/api_v0.yml#L668) and [Tag](https://github.com/forem/forem-docs/blob/b5ffacaf77b9ed09df7e2d6029fd25d17c246b7c/api_v0.yml#L987) is different and therefore I have added both.